scripts/dataset_processing/tts/sfbilingual/get_data.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+import random
+import subprocess
+from pathlib import Path
+
+import numpy as np
+from nemo_text_processing.text_normalization.normalize import Normalizer
+from opencc import OpenCC
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description='Prepare SF_bilingual dataset and create manifests with predefined split'
+    )
+
+    parser.add_argument(
+        "--data-root",
+        type=Path,
+        help="where the dataset will reside",
+        default="./DataChinese/sf_bilingual_speech_zh_en_vv1/SF_bilingual/",
+    )
+    parser.add_argument(
+        "--manifests-path", type=Path, help="where the resulting manifests files will reside", default="./"
+    )
+    parser.add_argument("--val-size", default=0.005, type=float, help="eval set split")
+    parser.add_argument("--test-size", default=0.01, type=float, help="test set split")
+    parser.add_argument(
+        "--seed-for-ds-split",
+        default=100,
+        type=float,
+        help="Seed for deterministic split of train/dev/test, NVIDIA's default is 100",
+    )
+
+    args = parser.parse_args()
+    return args
+
+
+def __process_transcript(file_path: str):
+    # Create zh-TW to zh-simplify converter
+    cc = OpenCC('t2s')
+    # Create normalizer
+    text_normalizer = Normalizer(
+        lang="zh", input_case="cased", overwrite_cache=True, cache_dir=str(file_path / "cache_dir"),
+    )
+    text_normalizer_call_kwargs = {"punct_pre_process": True, "punct_post_process": True}
+    normalizer_call = lambda x: text_normalizer.normalize(x, **text_normalizer_call_kwargs)
+    entries = []
+    i = 0
+    with open(file_path / "text_SF.txt", encoding="utf-8") as fin:
+        for line in fin:
+            content = line.split()
+            wav_name, text = content[0], "".join(content[1:])
+            wav_name = wav_name.replace(u'\ufeff', '')
+            wav_name = wav_name.replace('DL', 'SF')  # TODO, WAR: change DL to SF
```

Comment:
Does this `TODO` need to be addressed soon? Not quite sure which pattern is used for the final replace.

Reply:
It's a mistake of dataset itself. Wav's name is like com_SF_0001.wav, however, in the label file they offered, wav's name is com_DL_00001. When they fix it in the dataset, we could remove this WAR here.

Reply:
are they working on the fix? if not, could you please add such detailed comments there? Thanks.

Reply:
I would let them know. For now, add more details in the comments.